sdrbench/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(sdrbench_SOURCES
     test_ft8.cpp
     test_callsign.cpp
     test_ft8protocols.cpp
+    ../ft8/pack0.cpp
```

Comment:
This should not be added here. It is part of the FT8 library see `set(sdrbench_FT8_LIB ft8)` above...

Reply:
Have removed it, as doesn't seem to be needed now. I added it when you made the first FT8 related commit - maybe something has changed since then or was just a dodgy build that made it look like it was needed when it wasn't.

Reply:
Yes indeed there was a failing build that I corrected in a later commit.